src/Components/Search/SearchBar.tsx

```diff
@@ -141,6 +140,7 @@ export class SearchBar extends Component<Props, State> {
         } else if (performanceStart && sd.VOLLEY_ENDPOINT) {
           this.reportPerformanceMeasurement(performanceStart)
         }
+        const { viewer } = this.props
```

Comment:
Here, `this.props` has the just-fetched data (which is what we want). Before, `viewer` was stale and was referring to the previous fetched data.